hikari/api/entity_factory.py

```diff
@@ -563,6 +755,9 @@ def deserialize_channel(
     ) -> channel_models.PartialChannel:
         """Parse a raw payload from Discord into a channel object.
 
+        !!! note
+            This also deserializes to thread channels.
```

Comment:
Should we separate threads from old school channels?

Reply:
You can't separate them out, fetch_channel returns both threads and normal channels

Reply:
Ah, all good then :+1: